index.html

```diff
@@ -49,180 +49,224 @@ <h2 class="hex connect">LET’S CONNECT</h2>
                 </li>
-                </ul>
-                <div class="see-project">
+            </ul>
+            <div class="see-project">
                 <button type="button" class="hex">Get my resume</button>
-                </div>
-                <div class="languages-list">
+            </div>
+            <div class="languages-list">
                 <ul>
-                <li class="lang-title">
-                <p class="langusge">Languages</p>
-                <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
-                </li>
-                <li class="lang">
-                <img src="images/langusges/js.png" alt="js" />
-                <p>JavaSript</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/html.png" alt="js" />
-                <p>HTML</p>
-                </li>
-                <li class="lang">
-                <img src="images/langusges/css.png" alt="js" />
-                <p>CSS</p>
-                </li>
-                <li class="frameworks">
-                <p class="langusge">Frameworks</p>
-                <p class="sign">></p>
-                </li>
-                
-                <li class="frameworks">
-                <p class="langusge">Skills</p>
-                <p class="sign">></p>
-                </li>
+                    <li class="lang-title">
+                        <p class="langusge">Languages</p>
+                        <img class="allow-down" src="images/Icons/vector.png" alt="allow-down" />
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/js.png" alt="js" />
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/html.png" alt="js" />
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="images/langusges/css.png" alt="js" />
+                        <p>CSS</p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Frameworks</p>
+                        <p class="sign"></p>
+                    </li>
+                    <li class="frameworks">
+                        <p class="langusge">Skills</p>
+                        <p class="sign"></p>
+                    </li>
                 </ul>
-                </div>
             </div>
-
+        <section class="contact-me"> 
+            <div class="contact-background">
+                <h2>Contact me</h2>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need to build or a project that needs coding. I'd love to help with
+                </p>           
+                <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] like the previous reviewer noted, your form should submit successfully. Please kindly register with the [formspree service ](https://formspree.io/html/) so you can get your hash_id and update your form action `(example action='https://formspree.io/f/mjvdwrry')`. your form does not submit. this is how it works 👇🏾 

![Screenshot (941)](https://user-images.githubusercontent.com/65298048/230040988-2f01317c-9433-480e-978e-3fb8d53f8ad1.png)